docs/csharp/whats-new/csharp-14.md

```diff
@@ -0,0 +1,47 @@
+---
+title: What's new in C# 14
+description: Get an overview of the new features in C# 14. C# 14 ships with .NET 10.
+ms.date: 02/04/2025
+ms.topic: whats-new
+---
+# What's new in C# 14
+
+C# 14 includes the following new features. You can try these features using the latest [Visual Studio 2022](https://visualstudio.microsoft.com/vs/preview/) version or the [.NET 10 SDK](https://dotnet.microsoft.com/download/dotnet):
+
+- [`nameof` supports unbound generic types](#unbound-generic-types-and-nameof)
+
+C# 14 includes the [`field`](#the-field-keyword) contextual keyword as a preview feature.
```

Comment:
This is just a normal preview feature I think. Not sure why it isn't just part of the list. We expect to release it in C# 14.